docs/remote-mcp-server.md

```
@@ -0,0 +1,242 @@
+- `jwksUrl` を省略すると JWKS 検証が行われず常に失敗する
+- `issuer` は `${origin}/api/auth`（better-auth の basePath 付き）
+- `audience` は `${origin}`（`validAudiences` に登録した値）
+
+### テナントスコープ（暫定）
+
+- token から `userId` → `members` テーブルでユーザーの最初の org を取得
+- **TODO**: OAuth scope に `org:{orgId}` を含めて明示的にスコープ指定
+
+## ファイル構成（実装済み）
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)